cmd/gaia/Makefile

```diff
@@ -5,7 +5,6 @@ VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
 COMMIT := $(shell git log -1 --format='%H')
 LEDGER_ENABLED ?= true
 GOBIN ?= $(GOPATH)/bin
-GOSUM := $(shell which gosum)
```

Comment:
Question: Does this mean we can delete gosum?

Reply:
It is actually removed in this PR: https://github.com/cosmos/cosmos-sdk/pull/4262/files#diff-3c05cd732c157f5c2215e7026d1f8b23